src/App/Application.cpp

```diff
@@ -710,7 +678,7 @@ Document* Application::openDocument(const char * FileName, DocumentCreateFlags c
 }
 
 Document *Application::getDocumentByPath(const char *path, PathMatchMode checkCanonical) const {
-    if(!path || !path[0])
+    if(!path || path[0] == '\0')

```

Comment:
We recently added Base::Tools::equalsOrEmpty(path) which can be used instead.
I’m on my phone so I’m probably off on the exact name.

Reply:
`isNullOrEmpty()`